vue/src/App.vue

```diff
@@ -8,11 +8,15 @@ const isLoggedIn = ref(false);
 // Check if the user is logged in before rendering any UI. If they're not,
 // redirect them to the login page.
 onBeforeMount(async () => {
-  try {
-    await fetch('/api/status/', { redirect: 'error' });
+  if (import.meta.env.PROD) {
```

Comment:
Because Vite env vars are set at docker image build time, I don't think this will work.

Reply:
I think you're new authentication for the POST modelrun/sitemodel endpoints makes this moot anyways now.  I'll adjust this and the server settings.